build-packages/scandipwa-extensibility/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "@scandipwa/scandipwa-extensibility",
     "description": "Extension logic used by CSA.",
-    "version": "2.0.0",
+    "version": "2.0.1",
```

Comment:
Same here, do NOT bump versions manually!

Reply:
Reverted